qiskit/transpiler/coupling.py

```diff
@@ -407,6 +407,10 @@ def __str__(self):
             string += "]"
         return string
 
+    def __len__(self):
+        """Return the amount of edges in the coupling graph."""
+        return len(self.get_edges())
```

Comment:
You can just return `self.graph.num_edges()` here and get the edge count that way. It'll be more efficient since it just returning the internal count from rust instead of building an iterator of endpoint tuples and getting the length of that. That being said I'm not sure we want to have a `__len__` be edge count. I think most people would want to do `len(coupling_map)` and get the number of qubits/number of edges. In retworkx `__len__` returns the node count too.

Reply:
Is this `__len__` actually used / needed?  I couldn't see anywhere it's necessary in the code.